language/German/strings.po

```diff
@@ -850,27 +856,27 @@ msgstr "Drucker"
 #: src/frontend/mame/ui/info.cpp:55
 msgctxt "emulation-feature"
 msgid "magnetic tape"
-msgstr ""
+msgstr "Magnetband"
 
 #: src/frontend/mame/ui/info.cpp:56
 msgctxt "emulation-feature"
 msgid "punch tape"
-msgstr ""
+msgstr "Lochstreifen"
 
 #: src/frontend/mame/ui/info.cpp:57
 msgctxt "emulation-feature"
 msgid "magnetic drum"
-msgstr ""
+msgstr "Magnettrommel"
 
 #: src/frontend/mame/ui/info.cpp:58
 msgctxt "emulation-feature"
 msgid "solid state storage"
-msgstr ""
+msgstr "Festkörperspeicher"
```

Comment:
ugh. that sounds contrieved, but I have no better idea. Nobody will know what it means from that translation...

Reply:
Well, It IS the German translation for it...

Reply:
It is "a" German translation for it. Usually in the case of "solid state physics"/"solid state matter" as "Festkörperphysik" etc.

Note that I'm not against using it but I think the target audience wouldn't know what that meant. I don't know which systems use these media (and I can't check right now) but maybe "Flash Speicher" would be fine too? Although it's probably not strictly "Flash"?

Other than that I have no better idea at the moment *shrug*

Reply:
“Flash” is overly specific.  It’s used for any kind of solid state ROM/RAM media, from ROM sockets to SRAM cards to Flash storage to exotic things like core and rope memory.

Reply:
That's the reason why I chose that term in the first place.